Afterpay.podspec

```diff
@@ -1,23 +1,16 @@
-#
-#  Be sure to run `pod spec lint Afterpay.podspec' to ensure this is a
-#  valid spec and to remove all comments including this before submitting the spec.
-#
-#  To learn more about Podspec attributes see https://guides.cocoapods.org/syntax/podspec.html
-#  To see working Podspecs in the CocoaPods repo see https://github.com/CocoaPods/Specs/
-#
-
 Pod::Spec.new do |spec|
-  spec.name         = "Afterpay"
-  spec.version      = "0.0.1"
-  spec.summary      = "Afterpay iOS SDK"
-  spec.description  = <<-DESC
+  spec.name          = "Afterpay"
+  spec.version       = ENV['LIB_VERSION'] || '1.0.0' # fallback to major version
```

Comment:
That fallback seems a little sketchy but I can't think of a better way to do it.

Reply:
Yeah, it's unfortunate and should only be called in local contexts